nltk/text.py

```diff
@@ -173,36 +173,52 @@ def __repr__(self):
         return '<ConcordanceIndex for %d tokens (%d types)>' % (
             len(self._tokens), len(self._offsets))
 
-    def print_concordance(self, word, width=75, lines=25):
+    def print_concordance(self, word, width=80, lines=25, save=False):
         """
-        Print a concordance for ``word`` with the specified context window.
-
+        Print self._concordance
         :param word: The target word
         :type word: str
-        :param width: The width of each line, in characters (default=80)
-        :type width: int
         :param lines: The number of lines to display (default=25)
         :type lines: int
+        :param width: The width of each line, in characters (default=80)
+        :type width: int
+        :param save: The option to save the concordance.
+        :type save: bool
         """
         half_width = (width - len(word) - 2) // 2
-        context = width // 4 # approx number of words of context
+        context = width // 4  # approx number of words of context
 
+        # Find the instances of the word to create the ConcordanceLine
+        concordance_list = []
         offsets = self.offsets(word)
         if offsets:
```

Comment:
probably don't need the `if offsets` cause the loop just won't run if there are no offsets.